08-projects.Rmd

```diff
@@ -6547,17 +6547,17 @@ geo
 
 ---
 
-**Motivation:** We've done a lot of work with SQL this semester. Let's review concepts in this project and mix and match R, Python, and SQL to solve data-driven problems.
+**Motivation:** We've done a lot of work with SQL this semester. Let's review concepts in this project and mix and match R and SQL to solve data-driven problems.
```

Comment:
Although some 390 students probably still remember Python, I removed all the mentioning of Python as there is still limited material in the Examples Book on Python.

Reply:
Good idea